recipe/0001-loosen-pywin32-pin.patch

```diff
@@ -1,3 +1,4 @@
+Adapted from https://github.com/docker/docker-py/pull/2857/commits/2e8975698eb3bc36cb037e601ee798eace9bcc53
```

Comment:
Please add those to the recipe as a comment at the top of the patch line. Here they'll probably break the patching.

Reply:
Great point, I've moved the references.  But it turns out you can put anything you want at the top of a patch file, I believe it's ignored until it hits the `---` line.

Reply:
Ah. Yes! Still, we need to easily spot those during the next update to decide if we can drop them or not. So the best place is the recipe. Addition comments like, umerged, or merged and can be dropped next release also helps.